src/Base/PODVector.cpp

```diff
@@ -61,6 +61,9 @@ void make_PODVector(py::module &m, std::string typestr, std::string allocstr)
         .def(py::init<>())
         .def(py::init<std::size_t>(), py::arg("size"))
         .def(py::init<PODVector_type&>(), py::arg("other"))
+        .def("assign", [](PODVector_type & pv, T const & value){
+            pv.assign(pv.size(), value);
+        }, py::arg("value"), "assign the same value to every element")
```

Comment:
Overloaded because `vector.assign(vector.size(), 42)` to set a value is a bit cumbersome.

Reply:
Could also call this `set_value` in symmetry with `MultiFab`, please let me know what you prefer @WeiqunZhang @atmyers

Reply:
Waiting for https://github.com/AMReX-Codes/amrex/pull/3866